doc/level1.rst

```diff
@@ -1082,6 +1083,48 @@ Earlier choices are better than later ones.
    background. Set your ``accounts[addr].prefer_encrypt`` to
    ``nopreference`` and start sending Autocrypt headers.
 
+.. _enabling-disabling:
+
+Disabling Autocrypt
++++++++++++++++++++
+
+Once Autocrypt is enabled for a given account
+(``accounts[addr].enabled`` is set to ``true``), the user might choose
+to disable it.  By default, disabling should only set
+``accounts[addr].enabled`` to ``false``, and it SHOULD NOT destroy
+``accounts[addr].secret_key``.  This preserves the user's ability to
+read old encrypted e-mails, as well as being able to read encrypted
+e-mails that arrive after the user has disabled Autocrypt.
+
+The act of re-enabling Autocrypt after it was disabled SHOULD leave
+``accounts[addr].secret_key`` and ``accounts[addr].public_key``
+intact, so that the user continues using the same key.
+
+Destroying Secret Key Material
+++++++++++++++++++++++++++++++
+
+When disabling Autocrypt for an account, a Level 1 MUA MAY offer the
+user an opportunity to also destroy the secret key material for that
+account.  Since Autocrypt clients generally do not discuss secret keys
+with users, a MUA offering this choice should use a phrase like
+"destroy access to all encrypted messages", rather than referring to
+"keys" or "key material".
+
+A MUA that allows the user this opportunity SHOULD clearly indicate to
+the user that the destruction of this secret key material will leave
+them unable to read their previously-received encrypted messages as
+well as any new messages that arrive encrypted.  Note that for some
+users, this is a desirable feature: "destroy all messages" is an
+appropriate action to take in some circumstances.
+
+If the user selects this option, the MUA MUST clear both
+``accounts[addr].secret_key`` and ``accounts[addr].public_key``.
+
+After this key destruction has happened, the MUA is in a configuration
+where it could restart a :ref:`search for an Autocrypt Setup
+Message<setup-message-import>`.  For Level 1 MUAs, this is the
+simplest path for synchronizing secret keys between two MUAs that have
+mistakenly both enabled Autocrypt independently.
```

Comment:
I understand that you'd like to provide a path for users to deal with "MUA received an ASM but already has a secret" by way of saying "first destroy autocrypt secret, then re-scan for ASM". This might be useful in some situations but hurting others.  I suggest to remove this paragraph and leave the question how to deal with "dual" autocrypt state to the other PR (it can reference destruction of key state as stated in your PR if it needs to, no need to prescribe how we deal with dual autocrypt state here).

Reply:
OK, i've gone ahead and updated this branch to drop the controversial paragraph as well.  @hpk42 are you OK with merging this?